src/features/ajax/aggregate/gql.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @typedef {object} GQLMetadata
+ * @property {string} operationName Name of the operation
+ * @property {string} operationType Type of the operation
+ * @property {string} operationFramework Framework responsible for the operation
+ */
+
+/**
+ * @param {string|Object} gql The GraphQL object body sent to a GQL server
+ * @returns {GQLMetadata}
+ */
+export function parseGQL (gql) {
+  const contents = parseGQLContents(gql)
+  if (!contents) return
+  if (typeof contents !== 'object' || !contents.query) return
+
+  const operationName = contents.operationName || 'anonymous'
+  const operationType = contents.query.split(' ')[0]
+  return {
+    operationName, // the operation name of the indiv query
+    operationType, // query, mutation, or subscription,
+    operationFramework: 'GraphQL'
```

Comment:
Is L22 here really needed?

Reply:
i think if we some how centralize this behavior to be extraneous to just graphql (maybe do something similar for lightning framework for instance) it will be helpful to know the framework

Reply:
but for this implementation, i think it is not needed since its ignored in the batch processor.  Can remove if desired, but if theres ever a case we use the single parser it will be absent again